content/docs/command-reference/params/index.md

```diff
@@ -58,6 +58,10 @@ as the tree path to find those values. Supported types are: string, integer,
 float, and arrays (groups of params). Note that DVC does not ascribe any
 specific meaning to these values.
 
+> YAML 1.2 stores very large and very small numbers in scientific notation. Some
```

Comment:
I like that you specify a common problem here with scientific notation. In the note below, I like that you specify that PyYAML doesn't support YAML 1.2. Maybe you can combine them and have pretty much the same note in both places?

It would also be great to suggest solutions in both places (ruamel.yaml, JSON or other file formats).